src/components/Comment.tsx

```diff
@@ -0,0 +1,159 @@
+import React, { useState } from 'react';
+import { useForm } from 'react-hook-form';
+import { z } from 'zod';
+import { zodResolver } from '@hookform/resolvers/zod';
+import { useTranslations } from 'next-intl';
+import { useSession } from 'next-auth/react';
+import toast from 'react-hot-toast';
+import styled from 'styled-components';
+
+import { gql } from '../utils/gql';
+import { UserAnyKind } from '../../graphql/@generated/genql';
+import { backgroundColor, gray4, gray6 } from '../design/@generated/themes';
+
+import { Form } from './Form';
+import { FormCard } from './FormCard';
+import { FormTextarea } from './FormTextarea';
+import { UserPic } from './UserPic';
+import { Icon } from './Icon';
+import { Text } from './Text';
+import { FormAction } from './FormActions';
+import { Button } from './Button';
+import { Tip } from './Tip';
+import { Link } from './Link';
+
+interface CommentsCreateFormProps {
```

Comment:
`CommentProps` https://github.com/taskany-inc/issues/blob/main/docs/patterns.md#no-abstract-names-for-interfaces

Reply:
fix